src/qibolab/qubits.py

```diff
@@ -112,4 +106,7 @@ class QubitPair:
 
     qubit1: Qubit
     qubit2: Qubit
+
+    coupler: Optional[Coupler]
```

Comment:
```suggestion
    coupler: Optional[Coupler] = None
```
to simplify non-coupler platforms.